File: Online judge/Boj/Python/1120.py
a, b = input().split()
count, ans = 0, 1e9
for x in range(len(a), len(b)+1):
  count = 0
  new_b = b[x-len(a):x+len(a)]
  for y in range(len(a)):
    if new_b[y] is not a[y]:
      count+=1 
  ans = min(ans, count)
print(ans)

File: Online judge/Boj/C++/1120.cpp
#include <iostream>
#include <vector>
#include <algorithm>
#include <string>
#define endl '\n'

using namespace std;

int ans = 1e9;

int main() {
	ios::sync_with_stdio(0);
	cin.tie(0);
	
	string s1, s2;
	cin >> s1 >> s2;
	for (int i = 0; i < s2.size() - s1.size() + 1; i++) {
		int count = 0;
		for (int j = 0; j < s1.size(); j++) {
			if (s1[j] != s2[j+i]) {
				count++;
			}
		}
		ans = min(ans, count);
	}
	cout << ans << endl;
}